src/components/App.js

```diff
@@ -90,7 +91,7 @@ const App = () => {
           </Route>
 
           <Route path="/products/:productId">
-            <ProductView />
+            <ProductView selectProduct={setSelectedProduct}/>
```

Comment:
@crystaljoyce this should be 
<ProductView selectProduct={selectProduct} setSelectedProduct={setSelectedProduct} />